bookkeeper-server/src/main/java/org/apache/bookkeeper/client/LedgerMetadata.java

```diff
@@ -119,7 +119,7 @@ public LedgerMetadata(int ensembleSize,
         this.metadataFormatVersion = CURRENT_METADATA_FORMAT_VERSION;
 
         this.digestType = digestType.equals(BookKeeper.DigestType.MAC)
-            ? LedgerMetadataFormat.DigestType.HMAC : LedgerMetadataFormat.DigestType.CRC32;
+            ? LedgerMetadataFormat.DigestType.HMAC : LedgerMetadataFormat.DigestType.valueOf(digestType.toString());
```

Comment:
Can this be simplified with `this.digestType = LedgerMetadataFormat.DigestType.valueOf(digestType.toString())`?

Reply:
@sijie I wish, there is "HMAC" vs "MAC" case.

Reply:
gotcha